src/scss/_base.scss

```diff
@@ -1,21 +1,15 @@
 @import 'variables';
 @import 'mixins';
 @import 'helpers';
+@import 'tippy';
 
 @include box-sizing();
 
 .shepherd-element {
-  @include popper-theme-arrows($arrow-size: $arrow-size, $arrow-pointer-events: $arrow-pointer-events);
```

Comment:
I didn't think this change (moving the `@include popper-theme-arrows` invocation to `scss/tippy/_base-tooltips.scss`) was breaking any of the current behavior. 

Essentially, we're already applying `popper-theme-arrows` by default and making it unconfigurable. I think it'd be good to make an issue for addressing that in the future.

Reply:
It is configurable. All the variables passed to it can be overridden.

Reply:
Ah, true. But yeah, I think this can still be overridden by the user. See the way the [demo is doing exactly that](https://github.com/shipshapecode/shepherd/pull/261/files#diff-e1071944dd148ea7d2caef3b5fadfe44R124).

Reply:
The only difference is the need to target [Tippy's class/markup structure](https://atomiks.github.io/tippyjs/#creating-custom-themes) instead of `shepherd-element` directly.

I want to add more documentation around this and tweak some of the custom theming that Shepherd offers, but that feels like more of a [follow-up task](https://github.com/shipshapecode/shepherd/projects/2).

Reply:
Follow up is fine, but we need to support theming via some easy method before releasing 2.0.